OpenScienceJournal/whistlepunk_library/src/robolectricTest/java/com/google/android/apps/forscience/whistlepunk/devicemanager/NativeBleDiscovererTest.java

```diff
@@ -50,7 +51,7 @@ public void testConnectableSensor() {
             protected DeviceDiscoverer createDiscoverer(Context context) {
                 return new DeviceDiscoverer(context) {
                     @Override
-                    public void onStartScanning() {
+                    public void onStartScanning(ParcelUuid[] serviceUuids) {
```

Comment:
This test is failing on line 118. The number of sensors seen is 15 and the test thinks there should be just 1.

Reply:
15 is the correct value now, so I fixed the expected value in the test